life_expectancy/cleaning.py

```diff
@@ -1,13 +1,20 @@
 """Module cleaning, used to select a country and obtain clean data from
     the .tsv file in the data folder.
     """
+from dataclasses import dataclass
 import argparse
+import pathlib
+from typing import Callable
 import pandas as pd
 from pandas import DataFrame
-from life_expectancy.save_load import save_data, load_data
+from life_expectancy.save import save_data
+from life_expectancy.load import StrategyLoader, StrategyJSON,StrategyTSV
+from life_expectancy.regions import Country
 
-def clean_data(region_code:str, data_to_clean:DataFrame):
-    """This function receives data from the original .tsv file and cleans it for further
+CleaningStrategy = Callable[[Country, DataFrame], DataFrame]
+
+def clean_data_json(region_code:Country, data_to_clean:DataFrame):
```

Comment:
```suggestion
def clean_data_json(region_code:Country, data_to_clean:DataFrame) -> DataFrame:
```

Considering your defined type returns a DataFrame, your function should also return it or else mypy will complain.